File: Program diferents lenguages/Abstract factory/AbstractFactoryJava/src/VolvoDealer.java
 public class VolvoDealer extends AbstractFactory
    {
        @Override
        public  CAR createCAR()
        {
            return new VolvoCAR();
        }
        @Override
        public  TRUCK createTRUCK()
        {
            return new VolvoTRUCK();
        }

    }

File: DesignPattern_3_AbstractFactory/AbstractFactory/AbstractFactory/VolvoDealer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AbstractFactory
{
    public class VolvoDealer : AbstractFactory
    {

        public override CAR createCAR()
        {
            return new VolvoCAR();
        }

        public override TRUCK createTRUCK()
        {
            return new VolvoTRUCK();
        }

    }
}
